packages/express/BodyWrapper.ts

```diff
@@ -0,0 +1,36 @@
+import { Readable } from 'node:stream'
+import type { Request } from 'express'
+import type { NamedNode } from '@rdfjs/types'
+import type { Body } from '@kopflos-cms/core'
+import type { Environment } from '@rdfjs/environment/Environment.js'
+import type ClownfaceFactory from 'clownface/Factory.js'
+import type { Dataset } from '@zazuko/env/lib/DatasetExt.js'
+import type { DatasetFactoryExt } from '@zazuko/env/lib/DatasetFactoryExt.js'
+import onetime from 'onetime'
+
+export class BodyWrapper implements Body {
+  declare dataset: Promise<Dataset>
+
+  constructor(private readonly env: Environment<DatasetFactoryExt | ClownfaceFactory>, private readonly term: NamedNode, private readonly req: Readable & Pick<Request, 'quadStream'>) {
+    Object.defineProperty(this, 'dataset', {
```

Comment:
I'm curious why we need this dynamic construction. I tried avoiding it but struggled with `onetime`.